### PR TITLE
[Scoped] Update build script with no need define php 8.0 setup

### PR DESCRIPTION
**Pull request by samsonasik**
tested locally and it seems working ok.

### CODE REVIEW

File: full_build.sh [hunk 1]
```diff
@@ -23,7 +23,6 @@ composer install --ansi
 composer install --no-dev --ansi
 
 # early downgrade individual functions
-bin/rector process src/functions -c build/config/config-downgrade.php --ansi --clear-cache
 bin/rector process vendor/symfony/string/Resources/functions.php -c build/config/config-downgrade.php --ansi --clear-cache
```
Comment (TomasVotruba) [inline full_build.sh]: Why do you think this is not needed now?

Comment (samsonasik) [reply full_build.sh]: I copy it from latest `.github/workflows/build_scoped_rector.yaml`, the one that removed is `src/functions`, the  `vendor/symfony/string/Resources/functions.php` still in there.

https://github.com/rectorphp/rector-src/blob/cbbe410affb8afcfc3843e6738e882eef18a559d/.github/workflows/build_scoped_rector.yaml#L55-L56

Comment (samsonasik) [reply full_build.sh]: oh, I see that `src/functions` is prepended.

Comment (samsonasik) [reply full_build.sh]: updated to single run https://github.com/rectorphp/rector-src/pull/2449/commits/81f93fba874987497fc350807ab73fcb9d06048c

File: .github/workflows/build_scoped_rector.yaml [hunk 2]
```diff
@@ -52,8 +52,7 @@ jobs:
             -   run: composer install --no-dev --ansi
 
             # early downgrade individual functions
-            -   run: |
-                    bin/rector process src/functions vendor/symfony/string/Resources/functions.php -c build/config/config-downgrade.php --ansi
+            -   run: bin/rector process src/functions vendor/symfony/string/Resources/functions.php -c build/config/config-downgrade.php --ansi
```
Comment (samsonasik) [inline .github/workflows/build_scoped_rector.yaml:55]: `|` is not needed for single command

**Comment by TomasVotruba** [comment]
LGTM :+1:

**Pull request merged**